proxy/vm.go

```diff
@@ -16,14 +16,12 @@ package main
 
 import (
 	"encoding/binary"
-	"errors"
 	"fmt"
-	"math"
 	"net"
 	"os"
 	"sync"
 
-	hyper "github.com/hyperhq/runv/hyperstart/api/json"
+	"github.com/sameo/virtcontainers/hyperstart"
```

Comment:
afaik we are building hyperstart from https://github.com/clearlinux/hyperstart, it wont be any conflicts?